### ISSUE TITLE
Question of `logprob_fn` in the `squashed_gaussian`?

### ISSUE THREAD

**Issue by fuyw** [deepmind/rlax#54]
Hello, I have two questions about the `logprob_fn` in `squashed_gaussian`.

1. For a squashed multivariate Gaussian distribution: 
![image](https://user-images.githubusercontent.com/23493245/158076989-2719eab0-6020-45aa-80d1-afc4fe402605.png)

It seems there is a missing of 0.5 in `half_logdet = jnp.sum(jnp.log(sigma), axis=-1)`

https://github.com/deepmind/rlax/blob/87392e93a1da2fda240034930e801daf54590218/rlax/_src/distributions.py#L308

2. When computing the `log_det_jacobian`:

![image](https://user-images.githubusercontent.com/23493245/158077307-549dd42b-9469-456e-8e8f-66fda631dc06.png)

We need use `inv_transform(sample, action_spec)` to convert `a` to `u`.

https://github.com/deepmind/rlax/blob/87392e93a1da2fda240034930e801daf54590218/rlax/_src/distributions.py#L271

**Comment by elisejiuqizhang** [comment]
I second this, also curious to hear your reply

**Comment by katebaumli** [comment]
Thanks so much for the find, you will find the fix in #69 and further in #70 which replaces this with the more robust Distrax solution.
